### ISSUE TITLE
Company using atoum

### ISSUE THREAD

**Issue by jubianchi** [atoum/atoum.github.io#7]
Should we add the list of company using atoum?
In the Community page ?

**Comment by Grummfy** [comment]
Not so sure. One because the list will keep growing, second because it's
not do relevant.
Le 5 janv. 2016 12:34 AM, "Julien BIANCHI" notifications@github.com a
écrit :

**Comment by mikaelrandy** [comment]
Could be a promotional page, listing major company using atoum.

:+1: for me

**Comment by Hywan** [comment]
@jubianchi On the homepage! This is a **primary information**.

**Comment by Hywan** [comment]
@Grummfy We should maintain a top 25 for instance. The most prestigious ones.

**Comment by Grummfy** [comment]
well we could be inspirated by https://www.rust-lang.org/en-US/friends.html => see https://github.com/rust-lang/rust-www/blob/master/_data/users.yml & https://github.com/rust-lang/rust-www/blob/master/en-US/friends.html

**Comment by Grummfy** [comment]
I like the way of doing of rust. Not on the first page, but displayed randomly on a specific page.
Are we going in this direction?

**Comment by Hywan** [comment]
Yes :-). And the homepage can contain the most prestigious ones, it is important for the marketing.
